exercises/perfect-numbers/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "perfect_numbers"
+version = "0.0.1"
```

Comment:
given that https://github.com/exercism/problem-specifications/blob/master/exercises/perfect-numbers/canonical-data.json exists, unless there is a good reason to not do so, the test cases the Rust track uses should be those in that file, and this `version` line should be of the same version as https://github.com/exercism/problem-specifications/blob/master/exercises/perfect-numbers/canonical-data.json.

Reply:
I corrected the examples used in ceb59db. I did not use the invalid negative examples because the type of the input is u64, so they'll be invalid at compile-time. Version changed in Cargo.toml